run_l2monitor.sh

```diff
@@ -1,4 +1,19 @@
+getopts nv OPTION;
 while true
 do
-  node --unhandled-rejections=strict src/substrate/index.js || sleep 5
+  case ${OPTION} in
+  n)
+    node --unhandled-rejections=strict src/substrate/index.js n || sleep 5
+    ;;
+  v)
+    node --unhandled-rejections=strict src/substrate/index.js v | tee -a l2MonitorLogs || sleep 5
```

Comment:
Verbose mode has some defects:
This is a static file name. I don't think we should always write into a single name file (or append)
1. When we begin to call "sh run_l2montior.sh -v", we begin to write the log into a file like l2MonitorLogs_[startTimeStamp]
2. If we stop this bash runing and rerun "sh run_l2montior.sh -v", the log file should be into a new file.

3. I don't think the log content of verbose mode is valuable. I didn't find any valuable things (beside we record log in to log file) in verbose mode, compare with the normal mode.
Please think about it. When we use verbose mode, it normally mean we see some issue, some error, or monitor cannot continue in normal mode. (especially error happened when do verify) So we stop the monitor normal mode, and run it in verbose mode. Then what we need to show to help us to resolve the issue? At least, I think the 10 pending req contents should be recorded.

BTW, as we had reduced the log size I think the normal mode should also save logs in l2MonitorLogs_[startTimeStamp].